js/adapt-contrib-matching.js

```diff
@@ -15,6 +15,81 @@ define([
         "select2/dropdown/closeOnSelect"
     ], function(Utils, DropdownAdapter, AttachContainer, CloseOnSelect) {
 
+        // code added to make dropdown sit above body bottom where appropriate
```

Comment:
I think it would be nice to have the issue number listed here for future reference